src/odin/maneuversbuilder.cc

```diff
@@ -86,7 +86,10 @@ namespace valhalla {
 namespace odin {
 
 ManeuversBuilder::ManeuversBuilder(const Options& options, EnhancedTripLeg* etp)
-    : options_(options), trip_path_(etp) {
+    : options_(options), trip_path_(etp),
+      blind_mode_(options.costing_type() == Costing_Type_pedestrian &&
+                  options.costings().find(Costing::pedestrian)->second.options().transport_type() ==
+                      "blind") {
```

Comment:
ok so to look a bit more like the other types we should:

over in pedestriancost.cc we parse out kFook and kWheelchair, we should add kBlind
then in triplegbuilder we call on a trip edge set_pedestrian_type which stores this type on the edges of the path so we'll need to update the proto with that blind value as well (trip edge is a proto struct)